<ui version="4.0" >
 <class>buttoneditClass</class>
 <widget class="QDialog" name="buttoneditClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>按钮编辑</string>
  </property>
  <widget class="QPushButton" name="backpb" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>标签</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="biaoqian" >
   <property name="geometry" >
    <rect>
     <x>53</x>
     <y>10</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title" >
    <string>操作</string>
   </property>
   <widget class="QLineEdit" name="le_targetvar" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>60</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_weifanzhuan" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>90</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>位翻转</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_zhiweifuwei" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>10</y>
      <width>90</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>置位复位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>目标变量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_targetvar" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>60</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_yemiantiaozhuan" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>90</y>
      <width>90</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>页面跳转</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>120</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>目标页面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_targetpage" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>120</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_targetpage" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>120</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>240</y>
     <width>341</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title" >
    <string>颜色</string>
   </property>
   <widget class="QPushButton" name="pb_oncolor" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>110</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>ON颜色</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>70</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>指示变量</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_zhishidenkaiguan" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>40</y>
      <width>90</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>带指示灯的开关</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_kaiguan" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>90</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>开关</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>110</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>OFF颜色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_offcolor" >
    <property name="geometry" >
     <rect>
      <x>230</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_zhishivar" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_zhishivar" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>70</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_ziti" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>10</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>设置字体</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>40</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>标签颜色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_biaoqiancolor" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
